src/main/java/fi/otavanopisto/kuntaapi/server/integrations/vcard/VCardTranslator.java

```diff
@@ -0,0 +1,271 @@
+    }
+    
+    return Collections.emptyList();
+  }
+
+  private Address translateAddress(ezvcard.property.Address vCardAddress) {
+    if (vCardAddress == null) {
+      return null;
+    }
+    
+    Address result = new Address();
+    
+    result.setType(getAddressType(vCardAddress.getTypes()));
+    result.setPostOfficeBox(getString(vCardAddress.getPoBox()));
+    result.setPostalCode(getString(vCardAddress.getPostalCode()));
+    result.setPostOffice(getString(vCardAddress.getRegion()));
+    result.setStreetAddress(getLocalizedString(vCardAddress.getStreetAddress()));
+    result.setMunicipality(null);
+    result.setCountry(getString(vCardAddress.getCountry(), VCardConsts.DEFAULT_COUNTRY));
+    result.setQualifier(null);
+
+    return result;
+  }
+
+  private String getAddressType(List<AddressType> types) {
+    if (types != null && !types.isEmpty()) {
+      return types.get(0).getValue();
+    }
+
+    return null;
+  }
+
+  private String getPhoneType(List<TelephoneType> types) {
+    if (types != null && !types.isEmpty()) {
+      return types.get(0).getValue();
+    }
+
+    return null;
+  }
+
+  private List<LocalizedValue> getLocalizedString(String value) {
+    if (StringUtils.isNotBlank(value)) {
+      List<LocalizedValue> result = new ArrayList<>(1);
+      
+      LocalizedValue localizedValue = new LocalizedValue();
+      localizedValue.setLanguage(VCardConsts.DEFAULT_LANGUAGE);
+      localizedValue.setValue(value);
+      
+      result.add(localizedValue);
+      
+      return result;
+    }
+
+    return Collections.emptyList();
+  }
+
+  private String getOrganization(Organization organization) {
+    if (organization != null && organization.getValues() != null && !organization.getValues().isEmpty()) {
+      return organization.getValues().get(0);
+    }
+    
+    return null;
+  }
+  
+  private List<String> getOrganizationUnits(List<Organization> organizations) {
+    if (organizations != null && organizations.size() > 1) {
+      List<String> result = new ArrayList<>(organizations.size() - 1);
+      for (int i = 1, l = organizations.size(); i < l; i++) {
+        Organization organization = organizations.get(i);
+        String value = organization.getValues().isEmpty() ? null : organization.getValues().get(0);
+        if (StringUtils.isNotBlank(value)) {
+          result.add(value);
+        }
+      }
+      
+      return result;
+    }
+    
+    return Collections.emptyList();
+  }
+
+  private String getFirstName(StructuredName structuredName) {
+    if (structuredName != null && StringUtils.isNotBlank(structuredName.getGiven())) {
+      return structuredName.getGiven();
+    }
+    
+    return null;
+  }
+  
+  private String getLastName(StructuredName structuredName) {
+    if (structuredName != null && StringUtils.isNotBlank(structuredName.getFamily())) {
+      return structuredName.getFamily();
+    }
+    
+    return null;
+  }
+
+  private String getString(String value) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getString" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)